quadcopter_brain/src/quadcopter_brain/quadcopter_brain.py

```diff
@@ -67,12 +74,42 @@ def has_reached_waypoint(self, waypoint):
         print "Checking reached:"
         try:
             _, _, dist_from_waypoint = \
-                PositionTools.lat_lon_diff(self.current_lat,
-                                           self.current_long,
-                                           wpt_latitude,
-                                           wpt_longitude)
+                PositionTools.lat_long_diff(self.current_lat,
+                                            self.current_long,
+                                            wpt_latitude,
```

Comment:
wpt_latitude and wp_lat should be standardized throughout the code. pick one and use it everywhere